spell/spell-crafter-mainnet-workflow.md

```diff
@@ -33,221 +33,205 @@ Repo: https://github.com/makerdao/spells-mainnet
-    * [ ] [community](https://github.com/makerdao/community) repo commit hash corresponds to latest change
-    * [ ] Raw GitHub URL is correct
-    * [ ] Exec hash is correct (use `cast keccak -- "$(curl '$URL' -o - 2>/dev/null)"` where `wget` doesn't work)
+  * [ ] Run `make exec-hash` (or `make exec-hash date=YYYY-MM-DD`) and update spell code accordingly
+  * [ ] Ensure that executive vote file name and date is correct
+  * [ ] [community](https://github.com/makerdao/community) repo commit hash corresponds to latest change
+  * [ ] Raw GitHub URL is correct
+  * [ ] Exec hash is correct (use `cast keccak -- "$(curl '$URL' -o - 2>/dev/null)"` where `wget` doesn't work)
   * [ ] Ensure `description` date in `DssSpell.sol` matches target date inside Exec Doc
-* [ ] Make sure all review comments are either addressed or answered
-* [ ] Notify the reviewers
+* [ ] Make sure all review comments are either addressed or explicitly answered
+* [ ] Make sure all items in the Exec Sheet are confirmed, OTHERWISE notify Responsible Governance Facilitator
+* [ ] Notify the reviewers (e.g. "Exec Hash is added, reviews are addressed")
 
 ## Deployment Stage
 
 * [ ] Wait for at least two "good to deploy" comments (containing local tests) from the official reviewers
-* [ ] Pre-Deploy Setup and Checks (currently via `dapptools`)
-  * [ ] Set local env (`.sethrc`)
-    * [ ] Deployer
+* Pre-deploy setup and checks (currently via `dapptools`)
+  * Set local environment variables (`.sethrc`)
+    * Deployer
       * [ ] Avoid using the same deployer for mainnet and testnet (to avoid deploying contracts with the same address but different sources)
       * [ ] `export ETH_PASSWORD=~/.env/password.txt`
       * [ ] `export ETH_KEYSTORE=~/.ethereum/keystore`
       * [ ] `export ETH_FROM=<address>`
-    * [ ] Mainnet EIP1559
-      * [ ] `export ETH_GAS=X`
-        * [ ] Run `make estimate` and adjust `ETH_GAS` accordingly (e.g. 6000000)
-      * [ ] `export ETH_GAS_PRICE=$(seth --to-wei X gwei)`
-        * [ ] Check current gas price using `seth gas-price` and Set `ETH_GAS_PRICE` accordingly (e.g. 50 gwei)
-          * [ ] Consider adding margin to account for spikes (e.g. current gas price 25 `gwei`, 50 `gwei` could be set)
-      * [ ] `export ETH_PRIO_FEE=$(seth --to-wei X gwei)`
-        * [ ] Check [current gas priority fee](https://etherscan.io/gastracker) and set `ETH_PRIO_FEE` accordingly (e.g. 2 gwei)
-    * [ ] Mainnet RPC URL
-      * [ ] `export ETH_RPC_URL=<url>`
-    * [ ] Etherscan API KEY
-      * [ ] `export ETHERSCAN_API_KEY=<key>`
-    * [ ] Ensure to `source` the `.sethrc` file to make env vars available
-  * [ ] Check local env
+    * EIP1559
+      * [ ] Run `make estimate` to estimate gas
+      * [ ] `export ETH_GAS=X` with the output of the command above (e.g. `export ETH_GAS=6_000_000`)
+      * [ ] Check current gas price using `seth gas-price` and set `ETH_GAS_PRICE` accordingly (e.g. `50 gwei`)
```

Comment:
Wouldn't it be better top fetch it from the same source as the priority fee?

Reply:
It might be, but the aim of this PR is to just improve the structure or in some cases add/improve rules, but not change them, so I would put this out of scope 🙂